src/setup.py

```diff
@@ -0,0 +1,24 @@
+"""The Setup File for our Mailroom App."""
+
+
+from setuptools import setup
+
+setup(
+    name="Mailroom",
+    description="An implementation of the Mailroom Module",
+    version=0.1,
+    author=["Conor Clary", "Regenal Grant"],
+    author_email=["sclary50@gmail.com", "regenal@mac.com"],
+    license="MIT",
+    package_dir={'': './'},
+    py_modules=['main'],
+    install_requires=['numpy', 'faker', 'inquirer'],
```

Comment:
Are you actually using these packages? If not, remove them, because it takes time and space to install them.